py/trtorch/csrc/util.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <functional>
+#include <iostream>
+#include <string>
+#include "core/util/logging/TRTorchLogger.h"

```

Comment:
Just include "core/prelude.h" It has the macros too

Reply:
Done.